job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -53,6 +53,14 @@ def RHBOP_CURRENT_DROOLS_VERSION='8.29.0'
 // Should be uncommented and used with kogitoWithSpecDroolsNightlyStage once Current is set for RHPAM 7.14.0
 // def DROOLS_CURRENT_PRODUCT_VERSION= OPTAPLANNER_CURRENT_PRODUCT_VERSION
 
+// Drools Ansible Rulebook Integration
+def RULEBOOK_NEXT_PRODUCT_BRANCH='main'
+def RULEBOOK_NEXT_PRODUCT_CONFIG_BRANCH='master'
+
+def RULEBOOK_CURRENT_PRODUCT_VERSION='1.0.0'
+def RULEBOOK_CURRENT_PRODUCT_BRANCH='1.0.x'
+def RULEBOOK_CURRENT_PRODUCT_CONFIG_BRANCH='master'
```

Comment:
I suppose for the `RULEBOOK_CURRENT_PRODUCT_CONFIG_BRANCH` you will need a specific nightly branch? something like drools-ansible-integration/1.0.x ? or rulebook/1.0.x (as you have been calling it)

Reply:
yeah, you are right good catch

Reply:
I agree